lib/commentOnOpenedPullRequest.js

```diff
@@ -93,6 +102,32 @@ commentOnOpenedPullRequest._askAboutChanges = function (files, baseBranch) {
     return true;
 };
 
+commentOnOpenedPullRequest._askAboutContributors = function (userName, repositorySettings) {
+    if (!defined(repositorySettings.contributorsUrl)) {
+        return Promise.resolve(false);
+    }
+
+    return requestPromise.get({
+        url: repositorySettings.contributorsUrl,
+        headers: repositorySettings.headers,
+        json: true
+    })
+    .then(function (response) {
+        var contributorsFileContent = Buffer.from(response.content, 'base64').toString();
+        // Matches items that look like: [<string>](https://github.com/<username>)
+        // sets the username as the first capturing group.
+        var regex = /\[.+\]\(https:\/\/github.com\/(.*)\)/g;
```

Comment:
Maybe `github.com\/(\w+)`? It avoid matching things that aren't part of the username, and will still work with any format as long as they use a valid url.